config/locales/de.helpers.yml

```diff
@@ -76,19 +76,31 @@ de:
     have_fun: "Viel Erfolg beim Training!"
     closing: "%{appname} Team"
   match_mailer:
-    next_game: "dein Spiel (%{team_home} vs. %{team_away}) im Event %{name}"
+    start_time: "Wann? Am %{start_date} um %{start_time} Uhr"
+    place: "Wo? %{place}"
+    here: "Hier"
+    apply_changes_html: "%{match_link} kannst du die Änderungen vornehmen."
```

Comment:
To my mind, “%{match_link} kannst du Änderungen vornehmen.” (without “die”) would be a slightly better translation – however, this seems like a question towards our PO @svitlanakalancha.